tests/conftest.py

```diff
@@ -2333,6 +2333,22 @@ def empty_ge_cloud_data_context_config(
     return DataContextConfig(**data_context_config_dict)
 
 
+@pytest.fixture
+def ge_cloud_config_e2e() -> GeCloudConfig:
+    """
+    Uses live credentials stored in the Great Expectations Cloud backend.
+    """
+    base_url = os.environ["GE_CLOUD_BASE_URL"]
+    organization_id = os.environ["GE_CLOUD_ORGANIZATION_ID"]
+    access_token = os.environ["GE_CLOUD_ACCESS_TOKEN"]
+    ge_cloud_config = GeCloudConfig(
+        base_url=base_url,
+        organization_id=organization_id,
+        access_token=access_token,
+    )
+    return ge_cloud_config
```

Comment:
Should we x-fail if we don't have creds? Not 100% how to treat this.

Reply:
No, I think we fail if this test does not run well for any reason incl unreachable cloud. Although we may consider moving these e2e tests to a separate integration test pipeline that doesn't run on each PR.